src/service/resources-impl.js

```diff
@@ -1190,10 +1202,11 @@ export class Resources {
    * @param {number|undefined} newHeight
    * @param {number|undefined} newWidth
    * @param {boolean} force
-   * @param {function()=} opt_callback A callback function.
+   * @param {function()=} opt_callback A callback function
    * @private
    */
-  scheduleChangeSize_(resource, newHeight, newWidth, force, opt_callback) {
+  scheduleChangeSize_(resource, newHeight, newWidth, force,
+      opt_callback) {
     resource.resetPendingChangeSize();
     const layoutBox = resource.getLayoutBox();
     if ((newHeight === undefined || newHeight == layoutBox.height) &&
```

Comment:
will testing these separately and calling the callback with the right state resolve this?

Also i think there is a guard on ads js for this case. I agree that it would be ideal to solve/guard this here